Agent/SessionReplay/ViewCaptors/ViewDetails.swift

```diff
@@ -85,7 +85,7 @@ struct ViewDetails {
     // This function checks if there are any specfic masking rules assigned to a view. If it returns nils, the masking value will be assigned based on the value of the global based on it's type later.
     private static func checkIsMasked(view: UIView, viewName: String) -> Bool? {
         // Determine if this view should be masked
-        var agent = NewRelicAgentInternal.sharedInstance()
```

Comment:
Is there a reason you had to make this change? I know I made NewRelicAgentInternal.sharedInstance() nullable to fix a crash. Is it giving a warning or something? If the change is required Im good wit it.

Reply:
Ah can you please change this to 

```
        guard let agent = NewRelicAgentInternal.sharedInstance() else { return true }